dsn.go

```diff
@@ -95,8 +97,12 @@ func DSN(cfg *Config) (dsn string, err error) {
 	if cfg.Region != "" {
 		params.Add("region", cfg.Region)
 	}
-	if cfg.Authenticator != defaultAuthenticator {
-		params.Add("authenticator", strings.ToLower(cfg.Authenticator))
+	if cfg.Authenticator != AuthTypeSnowflake {
+		if cfg.Authenticator == AuthTypeOkta {
+			params.Add("authenticator", strings.ToLower(url.QueryEscape(cfg.OktaURL.String())))
```

Comment:
Do you need this QueryEscape? I thought Line 147 Encode escapes the parameter?

Reply:
Addressed